packages/flutter/test/material/tabs_test.dart

```diff
@@ -1007,14 +1008,14 @@ void main() {
               actions: SemanticsAction.tap.index,
               flags: SemanticsFlags.isSelected.index,
               label: 'TAB #0\nTab 1 of 2',
-              rect: new Rect.fromLTRB(0.0, 0.0, 108.0, 46.0),
+              rect: new Rect.fromLTRB(0.0, 0.0, 108.0, 48.0),
```

Comment:
We should probably include more of the constants from tabs.dart in material/constants.dart however for starters you could use kTextTabBarHeight here.